### ISSUE TITLE
DS - Checkbox and Checkbox Group Accessibility Review

### ISSUE THREAD

**Issue by kellylein** [department-of-veterans-affairs/va-mobile-library#430]
## Description 
As part of this story we want to:   

- Once the Checkbox Group component is available in Storybook, Brea will complete an accessibility review and create tickets for any additional work.

## Acceptance Criteria
  

-  [x] Review Checkbox component for and accessibility issues
-  [x] Review Checkbox Group component for and accessibility issues
-  [x] Create Bug tickets for any changes needed

## Accessibility Audit Checklist
- [x] Verify that the component scales with increased font sizes and with display zoom enabled.
- [x] Verify that the component passes color contrast minimums.
- [x] Verify that no screenreader issues exist.
- [x] Verify that the component works with a keyboard and that there are no keyboard traps.
- [x] Verify that the component works with Voice Access / Switch Control.
- [x] Verify that the overall structure is accessible (headings are properly labeled, etc.).
- [x] Verify that any error messaging displays properly / as expected.
- [x] Verify that any tappable areas (links, buttons, etc.) meet target size minimums.

**Comment by brea11y** [comment]
Findings were discussed [in this Slack thread](https://dsva.slack.com/archives/C07KFQ3U6QH/p1728328205127879). Two tickets were created based on the accessibility review.
